frontend/src/types/github.d.ts

```diff
@@ -16,6 +16,8 @@ interface GitHubUser {
 interface GitHubRepository {
   id: number;
   full_name: string;
+  stargazers_count?: number;
+  fromPublicRepoSearch?: boolean;
```

Comment:
Since we now extend this interface in the relevant component, we can remove the `fromPublicRepoSearch` property from here

Reply:
The fromPublicRepoSearch property has been removed from the base GitHubRepository type and moved to a component-specific GitHubRepositoryWithFlag interface. This ensures better type safety and prevents potential issues with accessing non-existent properties. (by OpenHands)